modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DatePicker/DatePicker.es.js

```diff
@@ -79,15 +79,13 @@ const getDelimiter = (dateFormat) => {
 };
 
 const useDateFormat = () => {
```

Comment:
If we're removing the `useMemo` hook, then the function `useDateFormat` isn't a custom hook anymore, more details [here](https://reactjs.org/docs/hooks-custom.html), and, ideally, we should remove the prefix `use` and name this function as `getDateFormat`, for instance.